src/einsteinpy/symbolic/ricci.py

```diff
@@ -85,6 +87,60 @@ def from_metric(cls, metric):
         rc = ChristoffelSymbols.from_metric(metric)
         return cls.from_christoffels(rc)
 
+    def change_config(self, newconfig="uu"):
+        """
+        Changes the index configuration(contravariant/covariant)
+
+        Parameters
+        ----------
+        newconfig : str
+            Specify the new configuration. Defaults to 'uu'
+
+        Returns
+        -------
+        ~einsteinpy.symbolic.metric.RicciTensor
+            New Ricci with new configuration. Defaults to 'uu'
+
+        Raises
+        ------
+        ValueError
+            Raised when new configuration is not 'll' or 'uu'.
+            This constraint is in place because we are dealing with Ricci Tensor.
+
+        """
+        if newconfig == self.config:
+            return self
+        elif newconfig == "uu" or newconfig == "ll":
+            inv_met = RicciTensor(
+                sympy.simplify(sympy.Matrix(self.arr.tolist()).inv()).tolist(),
+                self.syms,
+                config=newconfig,
+            )
+            inv_met._invmetric = self
+            return inv_met
+        else:
+            raise ValueError(
+                "Configuration can't have one upper and one lower index in Metric Tensor."
```

Comment:
This is a bothersome error as it is exceptionally common for tensors to use a mix of contravariant and covariant indices. Is this a result of @ritzvik's implementation of raising and lowering indices?

Reply:
Yes @cjayross , `change_config` function which @OVSofia wrote isn't right(cut her some slack, she is learning as well), but I was going to review it later.
This code was intended for metric tensor because a metric tensor with one contravariant and one covariant indices really doesn't make sense as it is effectively an Identity tensor Delta. 
You can see the jupyter example http://docs.einsteinpy.org/en/latest/examples/Playing_with_Contravariant_and_Covariant_Indices_in_Tensors(Symbolic).html to ascertain yourself of the fact that arbitrary index configuration does work.

Reply:
I see. I haven't looked at that link yet. Definitely good to see.

And you are correct regarding the Kronecker delta, however, the notion that the user supplying a metric with mixed indices should result in an error is not very good design.